dataengnews/news/views.py

```diff
@@ -1,3 +1,10 @@
 from django.shortcuts import render
```

Comment:
Looks like `render` is not used any where in this file. Please remove it.

Reply:
Done! :-)